proc-macros/tests/ui/correct/basic.rs

```diff
@@ -24,6 +24,13 @@ pub trait Rpc {
 	#[method(name = "array_params")]
 	async fn array_params(&self, items: Vec<u64>) -> RpcResult<u64>;
 
+	#[method(name = "rename_params", param_kind = map)]
```

Comment:
you shouldn't need "param_kind = map" because it's only used when you are using the proc macro API anyway it doesn't matter.

Reply:
thanks, removed it